backend/routes.py

```
@@ -35,6 +37,63 @@ def register_event_handlers():
+        async with httpx.AsyncClient() as client:
+            response = await client.get(
+                f"https://api.github.com/repos/{owner}/{repo}",
+                headers=headers,
+                timeout=10.0
+            )
+            
+            if response.status_code != 200:
+                raise HTTPException(
+                    status_code=response.status_code,
+                    detail=f"GitHub API error: {response.text}"
+                )
```

Ruff (0.14.11)

Abstract `raise` to an inner function

(TRY301)